notebooks/244-depth-estimation-videpth/244-depth-estimation-videpth.ipynb

```diff
@@ -0,0 +1,1078 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    ie = Core()
</pre><p>please rename ie to core, it may be not clear for new openvino users why you use this name and how it is connected with openvino runtime</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/978/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='25'/>

Reply:
I am renaming `ie` to `ie_core`. Hope so that is Ok.